packages/server/graphql/mutations/createGitHubTaskIntegration.ts

```diff
@@ -87,8 +87,8 @@ export default {
       )
     }
     // using teamMembers to get the preferredName as we need the members for the notification part anyways
-    const {preferredName: viewerName} = teamMembers.find(({userId}) => userId === viewerId)
-    const {preferredName: assigneeName} =
+    const {preferredName: viewerName} = teamMembers.find(({userId}) => userId === viewerId)!
```

Comment:
If `not-null` is not obvious directly from the code above, I would suggest using some check instead of `!`

Reply:
done!